ick/config/rules.py

```diff
@@ -82,6 +82,7 @@ class RuleConfig(Struct):
 
     command: Optional[str] = None
     data: Optional[str] = None
+    entry: Optional[str] = None
```

Comment:
It feels like command, data, and entry are all "the information to hand to the impl thing".  Do we need all three?  Are there times we use more than one of them together?

Reply:
The code and test now use `command`, so we don't need `entry` anymore?

Reply:
btw, maybe "payload" would be generic enough for this?